module-1/lab-api-scavenger-game/your-code/Challenge 2 .py

```diff
@@ -0,0 +1,7 @@
+url_inic = "https://api.github.com/repos/ironhack-datalabs/madrid-oct-2018/commits"
```

Comment:
Does this chunk of code works for you? I get an error in execution. Is this script your last version?
```Python 
flattened2 = json_normalize(results2)
KeyError: 'signature'
```